engine/standalone/app/src/universal/conf/application.conf

```diff
@@ -0,0 +1,8 @@
+standaloneConfig {
```

Comment:
I think that we need to add comment here, that purpose of this config is to handle some very basic case when someone mount own very simple model via volumes - model doesn't need any configuration. For more advanced usage, developer should write own config.